src/reduxStore.js

```diff
@@ -13,6 +14,8 @@ const persistConfig = {
 
 const rootReducer = combineReducers({
   transactions,
+  backupState: userReducer.backupState,
```

Comment:
backup and user management are different features, so we need to create separated reducers for them.
Within this story, you don't need to work with user-related features
The backup related state should look like this:
```js
{
  isInProgress: true/false
  error: ...
  lastbackupDate: timestamp
}
```

Reply:
Have discussed with Wen and split them into backupReducer and userReducer